docs/content/documentation/systems/Kernels/phase_field/KKSMultiACBulkF.md

```diff
@@ -1,8 +1,58 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # KKSMultiACBulkF
 !description /Kernels/KKSMultiACBulkF
 
+KKS Allen-Cahn kernel for the terms without a direct composition dependence.
+
+### Residual
+
+$$
+R=-\frac{dh}{d\eta}(F_a-F_b)+w\frac{dg}{d\eta}.
+$$
+
+### Jacobian
+
+#### On-diagonal
+
+We are looking for the $\frac \partial{\partial u_j}$ derivative of $R$, where
+$u\equiv\eta$. We need to apply the chain rule and will again only keep terms
+with the $\frac{\partial \eta}{\partial u_j}\frac{\partial}{\partial \eta}=\phi_j \frac{\partial}{\partial\eta}$
+derivative.
+
+$$
+\begin{eqnarray*}
+J &=& -\phi_j \frac{\partial}{\partial \eta}\left( \frac{dh}{d\eta}(F_a-F_b) \right) + w \phi_j \frac{\partial}{\partial \eta}\frac{dg}{d\eta} \\
```

Comment:
On-diagonal Jacobian: if the non-linear variable is $\eta_1$,
$$
J = \phi_j \left( \frac{d^2 h_1}{d \eta_1^2} F_1 + \frac{d^2 h_2}{d \eta_1^2} F_2 + \frac{d^2 h_3}{d \eta_1^2} F_3 + W_i \frac{d^2 g}{d\eta_1^2}  \right)
$$